### ISSUE TITLE
jing doesn't like rename not having the (optional) Xml:lang attribute

### ISSUE THREAD

**Issue by alexjbest** [rbeezer/mathbook#1019]
This is somewhat trivial but the deprecation for the the lang attribute of rename says that "@lang attribute of rename has been replaced by @xml:lang, *and is now optional if your document only uses one language*" (emphasis mine)

But jing complains if I don't list an xml:lang i.e

```
jing ../mathbook/schema/pretext.rng parity-pretty.xml
/Users/alex/parity/parity-pretty.xml:329:8: error: element "rename" missing required attribute "xml:lang"
```

**Comment by rbeezer** [comment]
Thanks, Alex.  Actually on a sticky note on my desk - I saw that go by recently 
on my own stuff.  I was going to attack it the past couple of days, but I've 
been distracted by some other (emergency) things for a bit.

The issue will help other know we are aware of it.

**Comment by rbeezer** [comment]
Fixed at 54f45af12ef0aee6f440cee776079251e62e604e.

Website may be updated later today (but not now).

Thanks, Alex.
